drivers/note/note_driver.c

```diff
@@ -580,8 +558,7 @@ static void note_record_taskname(pid_t pid, FAR const char *name)
       ti = (FAR struct note_taskname_info_s *)
             &g_note_taskname.buffer[g_note_taskname.head];
       ti->size = skiplen;
-      ti->pid[0] = 0xff;
-      ti->pid[1] = 0xff;
+      ti->pid = 0xffff;
```

Comment:
pid_t is defined as int in sys/types.h, so, shouldn't it be: ti->pid = INT_MAX; ?

Reply:
Yes, it needs to be modified here
But I think it would be better to change it to -1, which indicates an illegal pid

Reply:
change to INVALID_PROCESS_ID